src/components/serviceOfferings/ServiceOfferings.tsx

```diff
@@ -1,193 +1,49 @@
-    setSelectedProperty(event.target.value);
-    setIsPropertySelected(true);
-  };
-
-  async function handleSearch() {
-    setIsLoading(true);
-    const targetClass = selectedShape.shortSubject.replace('Shape', '');
-    const selfDescriptions = await ApiService.getSelfDescriptionsForShape(
-      authContext,
-      targetClass
-    );
-    const map = mapServiceOfferings(selfDescriptions.data);
-    console.log('Map:', map);
-    setSelfDescriptionData(map);
-    setIsLoading(false);
+  const search = (filter: string) => {
+    setSearchText(filter)
   }
 
   return (
-    <div className="service-offerings">
-      <header>
-        <h2>Service Offerings</h2>
-      </header>
-      {authContext.isAuthenticated && (
-        <div className="content">
-          <FormControl sx={{ minWidth: 200 }}>
-            <InputLabel id="shape-label">Select Shape</InputLabel>
-            <Select
-              labelId="shape-label"
-              id="shape-select"
-              value={selectedShape.shortSubject}
-              label="SHACL Shape"
-              onChange={handleShapeChange}
-            >
-              {shapes.map((shape) => (
-                <MenuItem key={shape.shortSubject} value={shape.shortSubject}>
-                  {shape.shortSubject}
-                </MenuItem>
-              ))}
-            </Select>
-          </FormControl>
-          {isShapeSelected && (
-            <FormControl sx={{ marginX: 1, minWidth: 200 }}>
-              <InputLabel id="property-label">Property</InputLabel>
-              <Select
-                labelId="property-label"
-                id="property-select"
-                value={selectedProperty}
-                label="Property"
-                onChange={handlePropertyChange}
-              >
-                {properties.map((property) => (
-                  <MenuItem key={property} value={property}>
-                    {property}
-                  </MenuItem>
-                ))}
-              </Select>
-            </FormControl>
-          )}
-          {isPropertySelected && (
-            <FormControl sx={{ minWidth: 200 }}>
-              <TextField
-                id="outlined-basic"
-                label="Keyword"
-                variant="outlined"
-              />
-            </FormControl>
-          )}
-          <div className="button">
-            {isShapeSelected && (
-              <Button variant="contained" onClick={handleSearch} size="large">
-                Search
-              </Button>
-            )}
-          </div>
-
-          <Padding key="i01" paddingTop="20px" />
-          <div>
-            {!isLoading &&
-              selfDescriptionData.length > 0 &&
-              selfDescriptionData.map((selfDescription) => {
-                return (
+    <>
+      <Header title={`${t('service-offerings.titles')} (${serviceOfferings.length} ${t('common.results')})`}/>
+      <Main>
+        <LoadingIndicator isLoading={state === 'LOADING'}/>
```

Comment:
Perhaps something for later. But we should implement a global LoadingIndicator so that we don't have to call it in multiple files

Reply:
Perhaps something for later. But we should implement a global LoadingIndicator so that we don't have to call it in multiple files